src/challenges.js

```diff
@@ -30,16 +56,48 @@ const cacheFunction = cb => {
 const reverseStr = str => {
   // reverse str takes in a string and returns that string in reversed order
   // The only difference between the way you've solved this before and now is that you need to do it recursivley!
+  if (str === "") 
```

Comment:
Instead of writing `if`/`else` like this, we can simply do this:

```JS
const reverseStr = (str) => {
  if (str) return reverseStr(str.substr(1)) + str[0];
  return str;
};
```